### ISSUE TITLE
question about  paper (Decoding Consistency Ld-)

### ISSUE THREAD

**Issue by yjl010228** [zxk1212/ConcealGS#1]
Dear author, this is a great piece of work！But I would like to seek your advice on some parts of the paper that I did not understand.
**Firstly**, as shown in the red box in the following image，I have doubts about this Ld- loss. In my opinion, I don't think it is necessary and practically we only upload the student model and check the student model? **Secondly**, I wonder that if all the training views from student model are check views. In other words, can I use the decoder to obtain hidden information for all student models' views? Thank you very much advanced!
![Image](https://github.com/user-attachments/assets/d69e76e0-1ef4-49f7-bc72-7ec5e1ba5793)

**Comment by LiuHengyu321** [comment]
Hi @yjl010228!

Thank you for your interest in our work!

(a) In ConcealGS, the Ld-loss is utilized to prevent the decoder from overfitting to the implicit information. We have observed that only giving implicit information to the decoder for recovering leads to overfitting: The decoder outputs the given image with any input. Thus Ld-loss is crucial for preventing the decoder from overfitting.
(b) In the setting of ConcealGS, only one view in training views is selected as check views.

**Comment by yjl010228** [comment]
> Hi [@yjl010228](https://github.com/yjl010228)!
> 
> Thank you for your interest in our work!
> 
> (a) In ConcealGS, the Ld-loss is utilized to prevent the decoder from overfitting to the implicit information. We have observed that only giving implicit information to the decoder for recovering leads to overfitting: The decoder outputs the given image with any input. Thus Ld-loss is crucial for preventing the decoder from overfitting. (b) In the setting of ConcealGS, only one view in training views is selected as check views.

Oh your explanation is very clear! I think I get it. Thank you again!
